arrows/vxl/high_pass_filter.cxx

```diff
@@ -0,0 +1,490 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "high_pass_filter.h"
+
+#include <arrows/vxl/image_container.h>
+#include <vital/util/enum_converter.h>
+
+#include <vil/vil_convert.h>
+#include <vil/vil_image_view.h>
+#include <vil/vil_math.h>
+#include <vil/vil_plane.h>
+#include <vil/vil_transpose.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace vxl {
+
+enum filter_mode
+{
+  MODE_box,
+  MODE_bidir,
+};
+
+ENUM_CONVERTER( mode_converter, filter_mode, { "box", MODE_box }, { "bidir",
+                                                                    MODE_bidir } )
```

Comment:
nitpick - maybe make the line break at the first `{` to make it more readable?

Reply:
Thanks. @mwoehlke-kitware for some reason `uncrustify` keeps changing this line for fairly reasonable formatting styles, including what Chet suggested. I've been trying to avoid commiting anything that uncrusify will change in the future. Is that a reasonable practice?

Reply:
I've been treating uncrustify as a recommendation, and committing things that are clearly correct even if they get changed. In this case, the issue seems to be with `nl_type_brace_init_lst` and is almost certainly a bug. Besides, what it's trying to do has the additional annoying property of being over length.

I haven't tried to minimize the config yet, but just those lines are minimal input for demonstrating the issue.